python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/abstract_interp_lib_gen.py

```diff
@@ -215,6 +215,12 @@ def aten〇lift_fresh_copy〡shape(self: List[int]) -> List[int]:
 def aten〇_log_softmax_backward_data〡shape(grad_output: List[int], output: List[int], dim: int, input_dtype: int) -> List[int]:
     return upstream_shape_functions.unary(grad_output)
 
+def aten〇isnan〡shape(self: List[int]) -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+def aten〇ne〇Tensor〡shape(self: List[int], other: List[int]) -> List[int]:
+    return upstream_shape_functions.unary(self)
```

Comment:
`AtenNeTensor` is a broadcast-able op. Shape function should look like:
```suggestion
    return upstream_shape_functions.broadcast(self, other)
```